client/deployment/src/main/resources/templates/libraries/microprofile/api.qute

```diff
@@ -47,13 +47,57 @@ public interface {classname} {
     {/if}{/for}
     {/if}{/for}
         {#if mutiny}
-            {#if return-response}
-    public io.smallrye.mutiny.Uni<jakarta.ws.rs.core.Response> {op.nickname}(
+            {#if mutiny-operation-ids.get(op.operationIdOriginal) == "Multi"}
```

Comment:
This is not for this PR, but eventually, we need to start thinking about sharding this template into use cases and reusable components to stop adding all these conditionals. It's getting hairy and will be hard to maintain.

Reply:
Somewhat. But it is not specific for this PR/use case. Something broader than that. Thanks for your time sending this PR!

Reply:
do you have any idea when we can expect the 2.4.8 ?

Reply:
There's only one PR left to merge, then we can do it.